pkg/ottl/parser.go

```diff
@@ -198,10 +200,10 @@ func (s *Statements[K]) Execute(ctx context.Context, tCtx K) error {
 		_, _, err := statement.Execute(ctx, tCtx)
 		if err != nil {
 			if s.errorMode == PropagateError {
-				err = fmt.Errorf("failed to execute statement: %w", err)
+				err = fmt.Errorf("failed to execute statement: %v, %w", statement.origText, err)
 				return err
 			}
-			s.telemetrySettings.Logger.Error("failed to execute statement", zap.Error(err))
+			s.telemetrySettings.Logger.Warn("failed to execute statement", zap.Error(err), zap.String("statement", statement.origText))
```

Comment:
Is it intentional that you're changing the log level here from `Error` to `Warn`?

Reply:
Yes.  My reasoning is that if using `error_mode: ignore` you are acknowledging that issues in individual statements should be ignored.  If we log them as `Error`, then any user who is monitoring their collector logs for errors would find these messages as errors.  I think `Warn` is a more appropriate level as it will let the users see that something happened, but not trigger any monitoring that is watching for errors.

Reply:
Yes, I agree with this reasoning.